packages/lime-app/Makefile

```diff
@@ -48,3 +48,53 @@ define Package/$(PKG_NAME)/postinst
 endef
 
 $(eval $(call BuildPackage,$(PKG_NAME)))
+#
+# Copyright (C) Libremesh 2017
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lime-app
+PKG_VERSION:= v0.2.0-alpha.5
```

Comment:
I am not sure if Makefile ignore it, but I would remove the space between = and v

Reply:
solved